azure/services/virtualmachines/mock_virtualmachines/client_mock.go

```diff
@@ -1,19 +1,3 @@
-/*
```

Comment:
Do we know what accounts for this?

Reply:
I accidentally committed before `make generate` finished running. Will fix this!